### ISSUE TITLE
Command does not display valid output "resin envs --application/device"

### ISSUE THREAD

**Issue by mkorakakis** [resin-io/resin-cli#120]
Doesn't return anything when using the --application and in the case of --device doesn't display the variable's name.

```
C:\Users\korakas\Documents\GitHub\resin-cli>node ./bin/resin envs --application demo
Request error: Internal Server Error

C:\Users\korakas\Documents\GitHub\resin-cli>node ./bin/resin envs --device 8d5eb924251b5ac05913e4a1d
46396b832e62b8e76733df78ed607cc5a50b9
ID  NAME VALUE
652      5555
```

**Comment by jviotti** [comment]
There are two different issues being mentioned here:

> C:\Users\korakas\Documents\GitHub\resin-cli>node ./bin/resin envs --application demo
> Request error: Internal Server Error

I'm unable to reproduce this. If the issue persists, please open a separate ticket where we can discuss the issue in depth.

> C:\Users\korakas\Documents\GitHub\resin-cli>node ./bin/resin envs --device 8d5eb924251b5ac05913e4a1d46396b832e62b8e76733df78ed607cc5a50b9
> ID  NAME VALUE
> 652      5555

I can reproduce this one. I'll send a PR fixing this soon.

**Comment by jviotti** [comment]
The device environment variables name issue was addressed in the SDK. This issue will be fixed once the PR is merged, the SDK published, and the SDK version updated in the CLI.
